tutorials/fileOutput/src/test/java/com/example/fileOutput/ApplicationTest.java

```diff
@@ -0,0 +1,97 @@
+package com.example.fileOutput;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+
+import javax.validation.ConstraintViolationException;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.hadoop.conf.Configuration;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.datatorrent.api.LocalMode;
+
+
+/**
+ * Test application in local mode.
+ */
+public class ApplicationTest {
+
+  private static final String baseDirName   = "target/fileOutput";
+  private static final String outputDirName = baseDirName + "/output-dir";
+  private static final File outputDirFile   = new File(outputDirName);
+
+  private void cleanup()
+  {
+    try {
+      FileUtils.deleteDirectory(outputDirFile);
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  // check that the requisite number of files exist in the output directory
+  private boolean check()
+  {
+    // Look for files with a single digit extension
+    String[] ext = {"0","1","2","3","4","5","6","7","8","9"};
+    Collection<File> list = FileUtils.listFiles(outputDirFile, ext, false);
+
+    return ! list.isEmpty();
+  }
+
+  // return Configuration with suitable properties set
+  private Configuration getConfig()
+  {
+    final Configuration result = new Configuration(false);
+    //conf.addResource(this.getClass().getResourceAsStream("/META-INF/properties.xml"));
```

Comment:
Maybe better to put all the constant values into a properties.xml so that users also have an example of that (will be needed to run on cluster anyways).